src/test/scala/com/goyeau/kubernetes/client/operation/CreatableTests.scala

```diff
@@ -26,22 +25,19 @@ trait CreatableTests[F[_], Resource <: { def metadata: Option[ObjectMeta] }]
       implicit client: KubernetesClient[F]
   ): F[Resource] =
     for {
-      _      <- NamespacesApiTest.createChecked[F](namespaceName)
       status <- namespacedApi(namespaceName).create(sampleResource(resourceName))
       _ = status shouldBe Status.Created
       resource <- getChecked(namespaceName, resourceName)
     } yield resource
 
   "create" should s"create a $resourceName" in usingMinikube { implicit client =>
-    createChecked(resourceName.toLowerCase, "some-resource")
+    createChecked(resourceName.toLowerCase, "create--resource")
```

Comment:
requires unique resource name to not clash with other test cases being running concurrently.

Reply:
Is the double dash `--` intended?

Reply:
no, some typo